test/benchmarks/driver_bench/src/main.mts

```diff
@@ -198,6 +199,27 @@ function calculateCompositeBenchmarks(results: MetricInfo[]) {
   return [...results, ...compositeResults];
 }
 
-const finalResults = calculateCompositeBenchmarks(results);
+function calculateNormalizedResults(results: MetricInfo[]): MetricInfo[] {
+  const primesBench = results.find(r => r.info.test_name === 'primes');
+  const pingBench = results.find(r => r.info.test_name === 'ping');
```

Comment:
this will be able to go away, then adjust the loop below to continue iff 'primes', and push `metrics.value / primesThroughput` for ping